HDSAviz/Basic_Plot_Updated.py

```diff
@@ -0,0 +1,130 @@
+"""This is a basic script for plotting an analysis file.
+
+    This script is just the first step in visualizing the data set. This file
```

Comment:
Isn't it assumed that all pieces of a given package (in this case HDSAviz) are dependent on each other?  That will be especially clear by the set of import statements at the top of each file, so I don't think we need to bloat the code with extra comments about it - just my opinion.  There will be a list of all the project's dependencies in the final readme and documentation for the whole program, so I don't think we need to add it to every file...